src/xye_script.cpp

```diff
@@ -1281,10 +1278,10 @@ void Load_Pit(TiXmlElement* el)
 /* Load Hint */
 void Load_Hint(TiXmlElement* el, bool warn)
 {
-    int c1=0,c2;
-        el->QueryIntAttribute("x",&LastX);
-        el->QueryIntAttribute("y",&LastY);
-        /*el->QueryIntAttribute("color",&c1);*/
+    //int c1=0,c2;
+    el->QueryIntAttribute("x", &LastX);
+    el->QueryIntAttribute("y", &LastY);
+    //el->QueryIntAttribute("color", &c1);
```

Comment:
Please remove the lines instead of just commenting them.

Reply:
I agree, but I kept it because it's needed for a chunk of commented out code later on. I didn't dare to remove the whole commented out section as it looks non-trivial. For this reason I'd prefer not to touch it. The commented out part can be removed in a separate clean-up ticket.